<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="20a47ccf-3a38-4dad-96e9-f0ab20773093" name="paytonli2014@ADMINIB-O91S1H8 2014-09-28 15:45:56" runUser="ADMINIB-O91S1H8\paytonli2014" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="4e5cae7b-75fd-4c06-ac5b-75de7e3bdad5">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="paytonli2014_ADMINIB-O91S1H8 2014-09-28 15_45_56">
      <DeploymentItem filename="C:\Workspace\Git\PTracker\src\PTracker\PTracker.Core\bin\Debug\PTracker.Core.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2014-09-28T15:45:56.9612003+08:00" queuing="2014-09-28T15:45:57.0102031+08:00" start="2014-09-28T15:45:57.0142033+08:00" finish="2014-09-28T15:45:57.2332158+08:00" />
  <ResultSummary outcome="Completed">
    <Counters total="1" executed="1" passed="1" error="0" failed="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="FileManagerConstructorTest" storage="c:\workspace\git\ptracker\src\ptracker\ptracker.core.test\bin\debug\ptracker.core.test.dll" id="79cefc58-5808-e29f-e99d-4c4f744a294c">
      <Execution id="dd928f6f-a8e8-4650-9055-4e21b5e2d868" />
      <TestMethod codeBase="C:/Workspace/Git/PTracker/src/PTracker/PTracker.Core.Test/bin/Debug/PTracker.Core.Test.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="PTracker.Core.Test.FileManagerTest, PTracker.Core.Test, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="FileManagerConstructorTest" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="79cefc58-5808-e29f-e99d-4c4f744a294c" executionId="dd928f6f-a8e8-4650-9055-4e21b5e2d868" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="dd928f6f-a8e8-4650-9055-4e21b5e2d868" testId="79cefc58-5808-e29f-e99d-4c4f744a294c" testName="FileManagerConstructorTest" computerName="ADMINIB-O91S1H8" duration="00:00:00.0175462" startTime="2014-09-28T15:45:57.0202037+08:00" endTime="2014-09-28T15:45:57.2212151+08:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="dd928f6f-a8e8-4650-9055-4e21b5e2d868">
      <Output>
        <StdOut>Loading file. path: C:\Workspace\Git\PTracker\src\PTracker\TestResults\paytonli2014_ADMINIB-O91S1H8 2014-09-28 15_45_56\Out
Error occuerd when loading file: C:\Workspace\Git\PTracker\src\PTracker\TestResults\paytonli2014_ADMINIB-O91S1H8 2014-09-28 15_45_56\Out. error: System.Exception: Error occuerd when loading file: C:\Workspace\Git\PTracker\src\PTracker\TestResults\paytonli2014_ADMINIB-O91S1H8 2014-09-28 15_45_56\Out. ---&gt; System.UnauthorizedAccessException: Access to the path 'C:\Workspace\Git\PTracker\src\PTracker\TestResults\paytonli2014_ADMINIB-O91S1H8 2014-09-28 15_45_56\Out' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at System.IO.File.OpenRead(String path)
   at PTracker.Core.FileManager.Load() in C:\Workspace\Git\PTracker\src\PTracker\PTracker.Core\FileManager.cs:line 58
   --- End of inner exception stack trace ---
FileManager constructed. path: C:\Workspace\Git\PTracker\src\PTracker\TestResults\paytonli2014_ADMINIB-O91S1H8 2014-09-28 15_45_56\Out</StdOut>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>